examples/simple/app.css

```diff
@@ -0,0 +1,110 @@
+
+/* mini reset */
+* { margin: 0; padding: 0; }
+
+body,
+html {
+  height: 100%;
+  overflow: hidden;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+header,
+footer {
+  height: 50px;
+  background: #eee;
+}
+
+#content {
+  position: relative;
+  flex: 1;
+}
+
+ul {
+  list-style: none;
+  padding: 0 0 0 0;
+  margin: 0;
+}
+
+ul section {
+  margin: 0;
+  padding: 0;
+}
+
+ul h2 {
+  position: sticky;
+  position: -webkit-sticky;
+  top: 0px;
+  height: 32px;
+
+  margin: 0;
+  padding: 0 6px;
+  box-sizing: border-box;
+
+  font-size: 0.9em;
+  line-height: 32px;
+
+  background: #efefef;
+  border-bottom: 1px #bcbcbc solid;
+
+  z-index: 100;
+}
+
+/*ul .background {
+  position: absolute;
+  width: 100%;
+
+  background: linear-gradient(to bottom, transparent 0px, transparent 45px, #EDEDED 45px, #EDEDED 70px, transparent 20px),
+              linear-gradient(to bottom, transparent 0px, transparent 15px, #EDEDED 15px, #EDEDED 25px, transparent 20px),
+              linear-gradient(to top, #bcbcbc, #bcbcbc 1px, white 1px, white);
+  background-position: 20% 88px, 5% -88px, 0 -88px;
+  background-size: 90% 88px, 60% 88px, 100% 88px;
+  background-repeat: repeat-y;
+
+  z-index: 0;
```

Comment:
maybe we can keep just the "line" part of the gradient (last background line, last position, last size (I think))